docs/source/build-with-javascript.md

```diff
@@ -269,7 +273,7 @@ DELETE artist FROM songs WHERE id = d754f8d5-e037-4898-af75-44587b9cc424;
 ```
 
 If you want to erase a specific column, you also should pass as parameter the `Clustering Key` and be very specific in which register you want to delete something. 
-On the other hand, the "normal delete" just need the `Partition Key` to handle it. Just remember: if you use the statement "DELETE FROM <table>" it will delete ALL the rows that you stored with that ID. 
+On the other hand, the "normal delete" just need the `Partition Key` to handle it. Just remember: if you use the statement `DELETE FROM <table>` it will delete ALL the rows that you stored with that ID. 
```

Comment:
The issue was here, <table> was being rendered as an HTML tag

Reply:
I never though about this possibility. Thanks for the fix!